docs/en/trust_marks.rst

```diff
@@ -219,12 +224,9 @@ The claims defined inside the TMs are compliant with the elements defined in the
       - String. The complete name of the service-supplying Entity.
       - |spid-icon| |cieid-icon|
 
-.. admonition:: |cieid-icon|
+.. warning::
 
-  In case of CIE id, the public Organizations having both the IPA code and a unique AOO code, MUST include the latter one in the claim id_code. In this case the claim **id_code** MUST include the IPA Code followed by the unique AOO code.
-  Furthermore, the value in the claim **exp** MUST NOT be greater than the duration of the specific 
-  conventions/agreements concluded in the onboarding process, between the Trust Mark issuer and the Organizations 
-  that receive the TM.
+  The value in the claim **exp** MUST NOT be greater than the duration of the specific agreements made in the onboarding process, between the Trust Mark issuer and the Organizations receiving the TM.
```

Comment:
~specific~

made -> submitted

in the -> during

Trust Mark issuer is a defined term?

Reply:
It is not a defined term, so I'll keep it in lower case if you agree. Trust Mark is a defined term, instead.